### PR TITLE
Project 2: Bulding with Responsive Design

### PR DESCRIPTION
**Pull request by taiwo2**
-This is the final project after the given feedback. Our project was all over the place so we redo it.
Some of the changes were:
-Added different stylesheets;
-Restructured index HTML;
- Added footer of the page;
Let us know what can be improved, and if approved inform us kindly.
Any feedback and assistance will be appreciated.

### CODE REVIEW

**Comment by descholar-ceo** [comment]
# This Pull Request is being Reviewed by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: :eyes: 
## Please wait a little bit ...

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1072 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="stylesheet/app.css" />
+
+    <link rel="stylesheet" href="stylesheet/more-info.css" />
+    <!-- <link rel="stylesheet" href="stylesheet/style-gitreset.css"> -->
```
Comment (descholar-ceo) [inline index.html]: Remove this line of commented code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1072 @@
...
+                role="img"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 320 512"
+              >
+                <path
+                  fill="currentColor"
+                  d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
+                ></path>
+              </svg>
+            </div> -->
```
Comment (descholar-ceo) [inline index.html]: This commented code is not needed to be left here

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @taiwo2 and @Leandro-Barretoo  :handshake: :handshake: :handshake: 
:+1: I liked that you have tried to use semantic HTML tags, that is very good, keep it up :muscle:
### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle:
## The project

- [ ] The getting started section of your readme file has some errors, learn about those errors below :point_down:
  - [ ] :x: This step `git checkout -b nextweb` is useless, since the readme file is normally displayed by default on the base branch, and it will be considered after you merge everything on the base branch, then there is no point of checkoing out another branch if a fellow programmer just wants to check if your project is cool.
  - [ ] :x: This step `git pull` doesn't make sense, since you already cloned the repository on the first step, then there is no point of pulling, by cloning a repository, all of the latest changes come, so please remove that `git pull` is not needed there.
- [ ] :warning: I like the last step of running the project with `live-server`, but the problem with it is this `localhost:3000`, any almost cases the `vs-code liver-server` is opened on port `5500`, so there is no need to specify the port there unless you have a setup which enforce it, please remove the port number, it might not work with all users and that will make your repository looks unprofessional
- [ ] Your readme file doesn't have a project description, please add it, it will make your repository outstanding and professional
- [ ] And also add a live demo link to your readme file
- [ ] The navbar is distorted, font-size should be decreased to make it looks like the original page
- [ ] The top three articles section:
  - [ ] All of boxes should have a background images and make sure that they look like the original one by using same font-size, and text-colors
  - [ ] The TNW logo should look like the original one and disapear starting from screen size 1024px and downward
- [ ] The `Latest news >` sction:
  - [ ] Please reduce the space which is in-between each component and reduce the size of cards, and make sure that it's centered
  - [ ] It should be showing three cards per row on medium screens, currently it's showing four (medium screens are between 768px and 1024px)
- [ ] `Latest funding rounds on Index.co > `:
  - [ ] It doesn't follow the original page, => make sure that every card is designed the same as the original
  - [ ] It should show for cards on large screens, and three card on medium screen, and and one card on mobile screen
  - [ ] Each card should have a cover image and title placed below the image
   - [ ] on mobile versions each card should have a cover image on the left side and title on the right side of the row
- [ ] The categories (`Hard Fork >`, Apps, Gear, Tech, etc) section:
  - [ ] Should be designed in such a way that it contains a header with the section name
  - [ ] Make sure that they look like the original page
- [ ] You are missing the `Latest deals section` please add it
- [ ] The footer touches each sides (left side and right side)
- [ ] And make sure that the footer is similar to the original one with the background colors and font-sizes

## The codes

- [ ] You have used `SVG` in your HTML file, that is okay, but it is not good since it has some side effect such difficulty of your code readability, a big chunk of crap right in the document, and inability to cache. Please remove all SVG images from html file and put them in a separate folder and call them as images
- [ ] You have used some ID selectors in your CSS file => that is a bad practice, you should use class instead
- [ ] You have a lot of CSS files, that is likely to make you repeating yourself, and also it reduce your code maintaimability. I am not against using more than one CSS file but that depends on the project size, since this project is just as small as a one page, I don't see a reason to use a lot of CSS files which sounds like almost every section has its styling file. For example: you have these files: `latest.css, new-latest.css, categories.css, ...` You can't write a separate file just for each section, this might give you headache when it comes to debugging or updating something. Please all of files which are styling some specific sections, merge them into one file and make sure that you don't repeat yourself
  - [ ] You have some linting errors please fix them
  - [ ] Add more description on this Pull request, you should actually explain what you have done on this Pull request in summary
  - [ ] It would be very nice if you add a screenshot of your clone in the readme file

If anything seems to be unclear to you, comment on this Pull Request and mention me as [@descholar-ceo](https://github.com/descholar-ceo)

 | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] The README has a comment that comes from a merge conflict, please remove it
image

- [ ] The link to the live preview should be an Actual link. To accomplish that use this syntax `[Text link](URL)`. Example: [@jubaan](https://www.github.com/jubaan)

- [ ] The README file has some conflicts please work on them.
image

- [ ] The linters are not set up correctly, they are not checking errors. It should appear stylelinter and webhint.
image

- [ ] Remove from repository the files that are not needed.
image
image

- [ ] The live preview link isn't working. It only shows a blank page.

- [ ] The index HTML has no code in it.

- [ ]  **Your readme file doesn't have a project description, please add it, it will make your repository outstanding and professional**

- [ ]  **And also add a live demo link to your readme file**

- [ ] Some changes that you have made previously have made the project to stop working as expected. The website is not displaying. index.html doesn't contain any code, the live preview link shows a white space, is not displaying anything this is because of the lack of code in the index.html

There are still some pending issues to be validated:
> * [ ]  The navbar is distorted, font-size should be decreased to make it looks like the original page
> * [ ]  The top three articles section:
>   
>   * [ ]  All of boxes should have a background images and make sure that they look like the original one by using same font-size, and text-colors
>   * [ ]  The TNW logo should look like the original one and disapear starting from screen size 1024px and downward
> * [ ]  The `Latest news >` sction:
>   
>   * [ ]  Please reduce the space which is in-between each component and reduce the size of cards, and make sure that it's centered
>   * [ ]  It should be showing three cards per row on medium screens, currently it's showing four (medium screens are between 768px and 1024px)
> * [ ]  `Latest funding rounds on Index.co > `:
>   
>   * [ ]  It doesn't follow the original page, => make sure that every card is designed the same as the original
>   * [ ]  It should show for cards on large screens, and three card on medium screen, and and one card on mobile screen
>   * [ ]  Each card should have a cover image and title placed below the image
>   * [ ]  on mobile versions each card should have a cover image on the left side and title on the right side of the row
> * [ ]  The categories (`Hard Fork >`, Apps, Gear, Tech, etc) section:
>   
>   * [ ]  Should be designed in such a way that it contains a header with the section name
>   * [ ]  Make sure that they look like the original page
> * [ ]  You are missing the `Latest deals section` please add it
> * [ ]  The footer touches each sides (left side and right side)
> * [ ]  And make sure that the footer is similar to the original one with the background colors and font-sizes
> 
> ## The codes
> * [ ]  You have used `SVG` in your HTML file, that is okay, but it is not good since it has some side effect such difficulty of your code readability, a big chunk of crap right in the document, and inability to cache. Please remove all SVG images from html file and put them in a separate folder and call them as images

- [ ]  **You have used some ID selectors in your CSS file => that is a bad practice, you should use class instead**

> * [ ]  You have a lot of CSS files, that is likely to make you repeating yourself, and also it reduce your code maintaimability. I am not against using more than one CSS file but that depends on the project size, since this project is just as small as a one page, I don't see a reason to use a lot of CSS files which sounds like almost every section has its styling file. For example: you have these files: `latest.css, new-latest.css, categories.css, ...` You can't write a separate file just for each section, this might give you headache when it comes to debugging or updating something. Please all of files which are styling some specific sections, merge them into one file and make sure that you don't repeat yourself
>   
>   * [ ]  You have some linting errors please fix them

- [ ]  **Add more description on this Pull request, you should actually explain what you have done on this Pull request in summary**

- [ ]  **It would be very nice if you add a screenshot of your clone in the readme file**

**COMMENTS: Before submitting for a re-review, check that all you project is working. To ensure this only push to the repository code that is needed and that you have personally check is working. Everything else leave it out of the repo. This way it would look more professional, tidy, and most of all you will ensure is working.**

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @taiwo2 and @Leandro-Barretoo  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
:+1: It's so amazing that you have done a great progress, your project is almost there
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] The clone now almost looks like the original page, the only remaining issue is collors and font-sizes. Your font-sizes are somehow bigger than the original ones, that is causing your clone to look strange, please try to adjust colors and font-sizes to make your clone good.
### Examples
#### 1. Navbars
| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-03 19-12-53 | Screenshot from 2020-12-03 19-13-55 |
:point_right: Could you see the difference :question: :question: :question: 

#### 2. `Latest funding rounds on Index.co ` section, it is even distorted on large screens, the three options should be aligned on the leftmost side of the screen

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-03 19-18-42| Screenshot from 2020-12-03 19-15-54|
:point_right: Could you see the difference :question: :question: :question: 

#### 3. `Categories (Apps, Gear, Tech, etc) section`

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-03 19-23-29| Screenshot from 2020-12-03 19-22-22|

:point_right: Could you see the difference :question: :question: :question: 

#### 4. Footer

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-03 19-25-52|Screenshot from 2020-12-03 19-26-43|

:point_right: Could you see the difference :question: :question: :question: 


- [ ] The pull request description is not yet worked on
- [ ] It's very good to have worked on readme file project description, but there some typos, instead of writing `The New Web` should be `thenextweb`
- [ ] The link to the live preview should be an Actual link. To accomplish that use this syntax [Text link](URL). Example: [@descholar-ceo](https://github.com/descholar-ce)
- [ ] You have used SVG in your HTML file, that is okay, but it is not good since it has some side effect such difficulty of your code readability, a big chunk of crap right in the document, and inability to cache. Please remove all SVG images from html file and put them in a separate folder and call them as images
- [ ] The footer should have two different colors, the one above and the other one below, yours has only one color, please fix that
- [ ] You have used some ID selectors in your CSS file => that is a bad practice, you should use class instead
- [ ] It would be very nice if you add a screenshot of your clone in the readme file
- [ ] You still have linters issues, please fix them

___
_If anything seems to be unclear to you, comment on this Pull request and mention me as [@descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

 | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @taiwo2 and @Leandro-Barretoo, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. You already fulfilled most of the requirements, kindly fix the following issues. 

[REQUIRED]

_Click the image to view it in full._

- [ ] Fix text overlay on the navigation. Use a higher `z-index` for the navigation.
- [ ] Fix the font family and font styles 
- [ ] The Categories (Apps, Gear, Tech, etc) row should have two articles per row

| Your Copy | Source |
| :---: | :---: |
|Screen Shot 2020-12-04 at 8 50 09 PM|Screen Shot 2020-12-04 at 9 06 43 PM|
|Screen Shot 2020-12-04 at 8 52 01 PM|

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. Kindly fix the issues stated in the previous review. 


breakpoints_requirement

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by arslanbisharat — APPROVED** [review]

**Pull request merged**